src/server/svr_jobfunc.c

```diff
@@ -6910,68 +6921,49 @@ recreate_exec_vnode(job *pjob, char *vnodelist, char *err_msg,
 								res_in_exec_vnode);
 
 						if (sel_entry > 0) {
-						/* there's already previous select/schedselect */
-						/* entry */
+						/* there's already previous select/schedselect entry */
 							if (pbs_strcat(
 								&new_select,
 								&ns_malloced,
 								"+") == NULL) {
-								log_err(-1, id, "pbs_strcat failed");	
+								log_err(-1, __func__, "pbs_strcat failed");	
 								goto recreate_exec_vnode_exit;
 							}
 						}
-						if (pbs_strcat( &new_select,
-							&ns_malloced,
-								"1") == NULL) {
-							log_err(-1, id,
-							"pbs_strcat failed");
+						if (pbs_strcat(&new_select, &ns_malloced, "1") == NULL) {
+							log_err(-1, __func__, "pbs_strcat failed");
 							goto recreate_exec_vnode_exit;
 						}
-						if (pbs_strcat( &new_select,
-							&ns_malloced,
-							    buf_sum) == NULL) {
-							log_err(-1, id,
-							"pbs_strcat failed");
+						if (pbs_strcat(&new_select, &ns_malloced, buf_sum) == NULL) {
+							log_err(-1, __func__, "pbs_strcat failed");
 							goto recreate_exec_vnode_exit;
 						}
 						if ((extra_res != NULL) && (extra_res[0] != '\0')) {
-							if (pbs_strcat(
-								&new_select,
-								&ns_malloced,
+							if (pbs_strcat(&new_select, &ns_malloced,
```

Comment:
Either combine the two lines of this this pbs_strcat() or break it at the '=='. I'd rather see what you're strcating on the same line.